gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/DatastoreReader.java

```diff
@@ -34,7 +34,7 @@
   /**
    * Returns an {@link Entity} for each given {@link Key} that exists in the Datastore.
    * The order of the result is unspecified.
-   * Results are loaded lazily therefore it is possible to get a {@code DatastoreException}
+   * Results are loaded lazily, so it is possible to get a {@code DatastoreException}
```

Comment:
Nit: I think this should either be on the above line or in a new paragraph (Also applies to the above line)